libraries/consul_config.rb

```diff
@@ -210,6 +211,7 @@ def params_to_json
           protocol
           reap
           read_replica
+	  raft_boltdb
```

Comment:
not to pick nits but is this spacing correct

Reply:
if ruby doesn't care i don't care

Reply:
you care though so i fixed it ❤️